.github/workflows/no-prs-to-translation.yml

```
@@ -16,10 +16,19 @@ jobs:
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
-            await github.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: 'We have implemented a new policy of no longer accepting curriculum-related PRs with changes to non-English files. We plan to implement a better way to translate the English challenges in the near future. We will make an announcement when it is ready. For now, please revert your changes to the non-English challenge files. Thank you.'
-            })
-            throw new Error('This PR appears to touch translated challenge files.')
+            if (context.payload.pull_request.head.repo.fork) {
+              await github.issues.createComment({
```

```suggestion
              github.issues.createComment({
```
Is this `await` necessary?  It looks like we're done once the comment has been created.

---

I believe it is - otherwise the action can potentially finish running before the API call to create the comment is resolved.